ledger/acctdeltas.go

```diff
@@ -1032,7 +1032,12 @@ func onlineAccountsNewRoundImpl(
 						prevAcct = updated
 					}
 				} else {
-					if prevAcct.AccountData.IsVotingEmpty() && newAcct.IsVotingEmpty() {
+					if prevAcct.AccountData.IsVotingEmpty() && newStatus != basics.Online {
```

Comment:
Isn't this inside the else block, started on line 1014, so we already know that `newStats != basics.Online`?

Reply:
true. but I guess this is more clear?